vulnerabilities/importers/apache_log4j.py

```diff
@@ -0,0 +1,252 @@
+import logging
+import xml.etree.ElementTree as ET
```

Comment:
Consider using the cyclonedx python library to parse this, and not parsing XML yourself